### PR TITLE
Fix schedule function deploy and delete

### PR DESCRIPTION
**Pull request by colerogers**
Fixes schedule function deploy & delete

### CODE REVIEW

File: src/deploy/functions/runtimes/discovery/v1alpha1.ts [hunk 1]
```diff
@@ -31,13 +37,22 @@ type EventTrigger = backend.EventTrigger & {
...
+};
+
 export type ManifestEndpoint = Base &
   backend.Triggered &
   Partial<backend.HttpsTriggered> &
   Partial<backend.CallableTriggered> &
   Partial<{ eventTrigger: EventTrigger }> &
   Partial<backend.TaskQueueTriggered> &
   Partial<backend.BlockingTriggered> &
+  Partial<{ scheduleTrigger: ScheduleTrigger }> &
```
Comment (taeold) [inline src/deploy/functions/runtimes/discovery/v1alpha1.ts:55]: Can we add a note on why we needed to make duplicate declartion of ScheduleTrigger (*Seconds vs *Duration)?

File: src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts [hunk 2]
```diff
@@ -307,6 +307,63 @@ describe("buildFromV1Alpha", () => {
       await expect(resolveBackend(parsed)).to.eventually.deep.equal(expectedBackend);
     });
 
+    it("copies v1 schedules", async () => {
```
Comment (taeold) [inline src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts:310]: Can we add 2 test cases? One that uses *Duration another that uses *Seconds?

Comment (taeold) [reply src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts:310]: Let's do this in a follow up PR.

**Review by taeold — APPROVED** [review]

File: src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts (hunk 2 above)
Comment (colerogers) [reply src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts:310]: There's already a test that uses *Seconds on L367

File: src/deploy/functions/runtimes/discovery/v1alpha1.ts (hunk 1 above)
Comment (colerogers) [reply src/deploy/functions/runtimes/discovery/v1alpha1.ts:57]: Added on L40

**Pull request merged**